### ISSUE TITLE
Animation problem in RTL Offcanvas

### ISSUE THREAD

**Issue by Viper-Bit** [coreui/coreui#273]
animation of rtl offcanvas have issue (just left and right offcanvas).

Browser: Chromium Edge
CoreUI: 4.0.5

codepen: [link](https://codepen.io/viper-bit/full/JjyRrbx)

fix:
transformx for rtl is wrong, solved with bellow css:
```css
html[dir='rtl'] .offcanvas-start {
  transform: translateX(100%);
}

html[dir='rtl'] .offcanvas-end {
  transform: translatex(-100%);
}

.offcanvas.show {
  transform: none !important;
}
```

**Comment by mrholek** [comment]
@Viper-Bit  Thank you, I will add this in the next release.

**Comment by Viper-Bit** [comment]
thx, fixed in version 4.1.0
